### ISSUE TITLE
createIsomorphicRequest change broke my setup in 4.11.0

### ISSUE THREAD

**Issue by RobZuazua** [clerkinc/javascript#1512]
@anagstef, this change broke my setup (AWS amplify serverless express lambdas)

I rolled back to previous version (4.10.15) and everything worked again.

```
{
    "errorType": "Runtime.UnhandledPromiseRejection",
    "errorMessage": "TypeError [ERR_INVALID_URL]: Invalid URL",
    "reason": {
        "errorType": "TypeError",
        "errorMessage": "Invalid URL",
        "code": "ERR_INVALID_URL",
        "input": "/api/s3/upload/0718/MP3",
        "stack": [
            "TypeError [ERR_INVALID_URL]: Invalid URL",
            "    at new NodeError (node:internal/errors:387:5)",
            "    at URL.onParseError (node:internal/url:565:9)",
            "    at new URL (node:internal/url:641:5)",
            "    at new Request (/var/task/node_modules/node-fetch-native/dist/shared/node-fetch-native.8afd3fea.cjs:6006:16)",
            "    at /var/task/node_modules/@clerk/clerk-sdk-node/dist/cjs/index.js:143:12",
            "    at createIsomorphicRequest (/var/task/node_modules/@clerk/backend/dist/index.js:1829:10)",
            "    at authenticateRequest (/var/task/node_modules/@clerk/clerk-sdk-node/dist/cjs/index.js:141:72)",
            "    at /var/task/node_modules/@clerk/clerk-sdk-node/dist/cjs/index.js:211:34",
            "    at Layer.handle [as handle_request] (/var/task/node_modules/express/lib/router/layer.js:95:5)",
            "    at next (/var/task/node_modules/express/lib/router/route.js:144:13)"
        ]
    },
    "promise": {},
    "stack": [
        "Runtime.UnhandledPromiseRejection: TypeError [ERR_INVALID_URL]: Invalid URL",
        "    at process.<anonymous> (file:///var/runtime/index.mjs:1186:17)",
        "    at process.emit (node:events:513:28)",
        "    at process.emit (node:domain:489:12)",
        "    at emit (node:internal/process/promises:140:20)",
        "    at processPromiseRejections (node:internal/process/promises:274:27)",
        "    at processTicksAndRejections (node:internal/process/task_queues:97:32)"
    ]
}
```

_Originally posted by @RobZuazua in https://github.com/clerkinc/javascript/issues/1393#issuecomment-1646676754_

**Comment by anagstef** [comment]
Hello @RobZuazua! Thank you for reaching out! We will look into it!

Can you let us know what nodejs version are you using?

**Comment by dimkl** [comment]
The issue seems to be related with this line : https://github.com/clerkinc/javascript/blob/4ff4b716fdb12b18182e506737afafc7dbc05604/packages/sdk-node/src/authenticateRequest.ts#L45

**Comment by dimkl** [comment]
Hello @RobZuazua and thank you for the feedback.
We have released the `@clerk/clerk-sdk-node@4.12.1` which should resolve the issue.
Feel free to re-open this if it's not fixed.

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
